public/electron.js

```diff
@@ -82,15 +82,35 @@ function initialize(queues) {
   )
 
   mainWindow.webContents.on('did-finish-load', () => mainWindow.show())
-  // mainWindow.webContents.openDevTools()
+
+  // Open devtools if DEV_TOOLS is set and has a value, presumably 1
+  if (process.env.PUBLISH50_DEV_TOOLS) {
+    mainWindow.webContents.openDevTools()
+  }
+
   mainWindow.on('closed', () => {
     mainWindow = null
   })
 }
 
 app.on('ready', async () => {
 
-  globalShortcut.register('CommandOrControl+Q', () => app.quit())
+  globalShortcut.register('CommandOrControl+Q', () => {
+
+    // Prompt user before closing the application if there are active jobs
+    if (Object.keys(queues['queues']['video transcoding']['childPool'].retained).length > 0 ||
+        Object.keys(queues['queues']['image processing']['childPool'].retained).length > 0 ||
+        Object.keys(queues['queues']['metadata']['childPool'].retained).length > 0 ||
+        Object.keys(queues['queues']['youtube']['childPool'].retained).length > 0) {
```

Comment:
Would that still work if we added a job queue tomorrow? Should we perhaps dynamically loop over all queues we have (which would probably be the keys in `queues['queues']`) and check if at least one of them has jobs that are running in which case we show the message?